### PR TITLE
Review compiler support

### PR DESCRIPTION
**Pull request by andre-richter**

### CODE REVIEW

File: src/compiler-support.md [hunk 1]
```diff
@@ -52,18 +55,18 @@ LLVM (http://llvm.org/):
...
-disabled (which is the case of AVR as of Rust 1.28) then you will need to modify the Rust source
+If LLVM supports the architecture you are interested in, but `rustc` is built with the backend
+disabled (which is the case of AVR as of Rust 1.28), then you will need to modify the Rust source
 enabling it. The first two commits of PR [rust-lang/rust#52787] give you an idea of the required
 changes.
 
 [rust-lang/rust#52787]: https://github.com/rust-lang/rust/pull/52787
 
-On the other hand, if LLVM doesn't support the architecture but a fork of LLVM does you will have to
-built `rustc` against the fork. The Rust build system allows this and in principle should just
```
Comment (japaric) [inline src/compiler-support.md]: Here I meant against the *LLVM* fork. I'm not sure "from the fork" conveys the right meaning. To me that sounds like you are building rustc from a fork of rustc, which could also be true, but it's not what we want to say here. We want to say that the LLVM component of rustc needs to be swapped.

**Comment by andre-richter** [comment]
Thanks for the feedback. Quite some confusion potential it seems. Is my new suggestion clearer now?

**Comment by japaric** [comment]
Yes, it's clearer now. Thanks!

bors r+

**Pull request merged**